conda_lsst/recipe_maker.py

```diff
@@ -84,7 +84,7 @@ def gen_conda_package(self, product, sha, eups_version, giturl, eups_deps, eups_
 		# the lsst-product-configs package as well, as that's where the .table
 		# and .cfg files are.
 		if depends_on_internal:
-			product_configs = 'lsst-product-configs ==0.0.1'
+			product_configs = 'lsst-product-configs ==1.0.1925'
```

Comment:
Add a key to `config.yaml` to read the version from (so we don't have to touch the code). E.g.
```
internal_product_configs_package: "lsst-product-configs ==1.0.1925"
```